charts/kyuubi/templates/_helpers.tpl

```diff
@@ -16,33 +16,18 @@
 */}}
 
 {{/*
-Expand the name of the chart.
+A comma separated string of enabled frontend protocols, e.g. "MYSQL,REST,THRIFT_BINARY".
```

Comment:
https://kyuubi.readthedocs.io/en/master/deployment/settings.html#frontend
<html>
<body>
<!--StartFragment-->
<table>
<tbody>
<tr>
<td>kyuubi.frontend.protocols</td>
<td>THRIFT_BINARY</td>
<td>A comma separated list for all frontend protocols <ul> <li>THRIFT_BINARY - HiveServer2 compatible thrift binary protocol.</li> <li>THRIFT_HTTP - HiveServer2 compatible thrift http protocol.</li> <li>REST - Kyuubi defined REST API(experimental).</li>  <li>MYSQL - MySQL compatible text protocol(experimental).</li>  <li>TRINO - Trino compatible http protocol(experimental).</li> </ul></td>
<td>seq</td>
<td>1.4.0</td>
</tr>
</tbody>
</table>

<!--EndFragment-->
</body>
</html>

Ok, I'll do! I keep learning Kyuubi and I see that frontends `REST`, `MYSQL` and `TRINO` marked as experimental.
Could you please give me some short info about them and their statuses, just for my learning?
Should I remove them all from the PR for now?

Reply:
Sorry for the late reply, it's during the Chinese Spring Festival holiday.

- THRIFT_BINARY is much much stable, w/ lots of production use cases.
- THRIFT_HTTP should work well w/ a single node, but may not be in the HA mode, since the HTTP is the short connection, and Kyuubi Server does not implement the shared session state across multi-instances.
- REST the batch API(the Livy-like API) should be considered as GA since 1.7, which fully supports HA mode and has production use cases in different companies. The rest of the API has a similar issue w/ THRIFT_HTTP in HA
- MYSQL 1)network protocol is implemented, 2)authentication does not work, 3)it does not support MySQL metadata queries which access the `information_schemas`. Basically, you can access Kyuubi using `mysql` cli or `mysql-connector-j` w/o authentication.
- TRINO it's under development, I guess it can work limited in 1.7